src/vendor.browser.ts

```diff
@@ -13,7 +13,7 @@ import 'rxjs/add/operator/mergeMap';
 
 // 3rd Party Libraries
 import 'jquery';
-import 'bootstrap';
 import 'lodash';
 import 'd3';
 import 'd3-tip';
+import 'ng2-bootstrap';
```

Comment:
I don't think we're using the original bootstrap import for anything, as all of our styles are compiled from the bootstrap sass files in the assets directory. It may be better to just try removing the bootstrap import, rather than replacing it with ng2-bootstrap.

I'm also not convinced this is the proper way to use ng2-bootstrap. I expected it to be imported within individual components just like the UI switch component. Maybe I'm wrong though. Either way, if we're not currently using any of its components, it should just be removed until its needed.